JitsiConference.js

```diff
@@ -1307,8 +1312,10 @@ JitsiConference.prototype.onIncomingCall
         // do not wait for XMPPEvents.PEERCONNECTION_READY, as it may never
         // happen in case if user doesn't have or denied permission to
         // both camera and microphone.
+        logger.info('Starting CallStats for JVB connection...');
         this.statistics.startCallStats(
-            this.jvbJingleSession.peerconnection, this.myUserId());
+            this.jvbJingleSession.peerconnection,
+            'jitsi' /* Remote user ID for JVB is 'jitsi' */);
```

Comment:
I agree and it was 'jvb' at the very beginning, but then Emil wanted to have it called 'jitsi'.